sdks/python/apache_beam/runners/interactive/dataproc/dataproc_cluster_manager.py

```diff
@@ -167,8 +166,7 @@ def create_flink_cluster(self) -> None:
             },
             'gce_cluster_config': {
                 'metadata': {
-                    'flink-start-yarn-session': 'true',
-                    'PIP_PACKAGES': 'apache-beam[gcp]=={}'.format(beam_version)
```

Comment:
Are we sure that as long as we provide the environment_config, the installation of a Python SDK on the cluster node is not needed?

Reply:
Yes, I did more testing and it seems that when we pass in the environment_config, the Python SDK does not need to be installed on the cluster node. This also greatly reduces the amount of time it takes to provision a new cluster.